tests/settings/e2e/steps/custom-formats.js

```diff
@@ -0,0 +1,89 @@
+// @flow
+import { Given, When, Then } from 'cucumber';
+import { expect } from 'chai';
+import { termsOfUseHelpers, chooseCustomOptionsByValue, getSelectedCustomOptions, getValueFromSelector, screenElementSelectors } from './helpers';
+import { timeout } from '../../../common/e2e/steps/helpers';
+import type { Daedalus } from '../../../types';
+
+declare var daedalus: Daedalus;
+
+Given(/^I choose the following custom formats:$/, async function(formatsTable) {
+  const chosenFormats = formatsTable.hashes();
+  const [
+    { value: numberValue },
+    { value: dateValue },
+    { value: timeValue },
+  ] = chosenFormats;
+  await chooseCustomOptionsByValue.call(this, numberValue, dateValue, timeValue);
+});
+
+Given(/^I have changed the following custom formats:$/, async function(formatsTable) {
+  const chosenFormats = formatsTable.hashes();
+  await this.client.executeAsync((chosenFormats, done) => {
+    Promise.all(
+      chosenFormats.map(({ param, value }) =>
+        daedalus.stores.profile._updateUserLocalSetting({ param: `${param}Format`, value }))
+    ).then(done);
+  }, chosenFormats);
+  await timeout(1500);
```

Comment:
@daniloprates is `timeout()` necessary here?

Reply:
✅